TiDB binlog: update descriptions about commit ts and passoword (#5986)
pingcap/docs: tidb-binlog/deploy-tidb-binlog.md

        [syncer.to]
        host = "192.168.0.13"
        user = "root"

>         # If you don’t want to set a cleartext `password` in the configuration file, you can create `encrypted_password` using `./binlogctl -cmd encrypt -text string`.

Reviewer: Abbreviations of this type are not recommended in technical documentation.
```suggestion
        # If you do not want to set a cleartext `password` in the configuration file, you can create `encrypted_password` using `./binlogctl -cmd encrypt -text string`.
```

>         # When you create `encrypted_password` and it is not empty, the `password` above will be ignored, because `encrypted_password` and `password` can’t take effect at the same time.

Reviewer: ```suggestion
        # When you have created an `encrypted_password` that is not empty, the `password` above will be ignored, because `encrypted_password` and `password` cannot take effect at the same time.
```
